lib/features/public_asset/presentation/clean_up_public_assets_interactor_bindings.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:get/get.dart';
+import 'package:jmap_dart_client/http/http_client.dart';
+import 'package:tmail_ui_user/features/base/interactors_bindings.dart';
+import 'package:tmail_ui_user/features/public_asset/data/datasource/public_asset_datasource.dart';
+import 'package:tmail_ui_user/features/public_asset/data/datasource_impl/remote_public_asset_datasource_impl.dart';
+import 'package:tmail_ui_user/features/public_asset/data/network/public_asset_api.dart';
+import 'package:tmail_ui_user/features/public_asset/data/repository/public_asset_repository_impl.dart';
+import 'package:tmail_ui_user/features/public_asset/domain/repository/public_asset_repository.dart';
+import 'package:tmail_ui_user/features/public_asset/domain/usecase/add_identity_to_public_assets_interactor.dart';
+import 'package:tmail_ui_user/features/public_asset/domain/usecase/clean_up_public_assets_interactor.dart';
+import 'package:tmail_ui_user/features/public_asset/domain/usecase/delete_public_assets_interactor.dart';
+import 'package:tmail_ui_user/features/public_asset/domain/usecase/remove_identity_from_public_assets_interactor.dart';
+import 'package:tmail_ui_user/main/exceptions/remote_exception_thrower.dart';
+import 'package:uuid/uuid.dart';
+
+class CleanUpPublicAssetsInteractorBindings extends InteractorsBindings {
+  @override
+  void dependencies() {
+    Get.lazyPut(
+      () => PublicAssetApi(Get.find<HttpClient>(), Get.find<Uuid>()),
+      tag: 'CleanUpPublicAssetsInteractorBindings');
```

Comment:
should be create const for it

Reply:
`HttpClient` is NOT `const`, hence can't

Reply:
I mean you should set the tag value to const

Reply:
Done